src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1 calendar__day--start-day-mon"></div>
+      <div class="calendar__day calendar__day--2 calendar__day--start-day-tue"></div>
+      <div class="calendar__day calendar__day--3 calendar__day--start-day-wed"></div>
+      <div class="calendar__day calendar__day--4 calendar__day--start-day-thu"></div>
+      <div class="calendar__day calendar__day--5 calendar__day--start-day-fri"></div>
+      <div class="calendar__day calendar__day--6 calendar__day--start-day-sat"></div>
+      <div class="calendar__day calendar__day--7 calendar__day--start-day-sun"></div>
+      <div class="calendar__day calendar__day--8"></div>
+      <div class="calendar__day calendar__day--9"></div>
+      <div class="calendar__day calendar__day--10"></div>
+      <div class="calendar__day calendar__day--11"></div>
+      <div class="calendar__day calendar__day--12"></div>
+      <div class="calendar__day calendar__day--13"></div>
+      <div class="calendar__day calendar__day--14"></div>
+      <div class="calendar__day calendar__day--15"></div>
+      <div class="calendar__day calendar__day--16"></div>
+      <div class="calendar__day calendar__day--17"></div>
+      <div class="calendar__day calendar__day--18"></div>
+      <div class="calendar__day calendar__day--19"></div>
+      <div class="calendar__day calendar__day--20"></div>
+      <div class="calendar__day calendar__day--21"></div>
+      <div class="calendar__day calendar__day--22"></div>
+      <div class="calendar__day calendar__day--23"></div>
+      <div class="calendar__day calendar__day--24"></div>
+      <div class="calendar__day calendar__day--25"></div>
+      <div class="calendar__day calendar__day--26"></div>
+      <div class="calendar__day calendar__day--27"></div>
+      <div class="calendar__day calendar__day--28 calendar__day--month-lenght-28"></div>
+      <div class="calendar__day calendar__day--29 calendar__day--month-lenght-29"></div>
+      <div class="calendar__day calendar__day--30 calendar__day--month-lenght-30"></div>
+      <div class="calendar__day calendar__day--31 calendar__day--month-lenght-31"></div>
+    </div>
```

Comment:
It's not good idea to set `n` for every block. 
Also check task description, the point is to set modificator `--start-day-{dayname}` for block `calendar`, not for calendar cells 
and it should change starting position(as on real calendar). The same with month-length-{n}, it should add display-none for children after  `n`